docs/csharp/tour-of-csharp/tutorials/index.md

```diff
@@ -8,10 +8,10 @@ ms.custom: mvc
 
 Welcome to the introduction to C# tutorials. These lessons start with interactive code
 that you can run in your browser. You can learn the basics of C# from the
-[C# 101 video series](https://aka.ms/dotnet3-csharp) before starting these interactive lessons.
+[C# for Beginners video series](https://aka.ms/dotnet/beginnervideos/youtube/csharp) before starting these interactive lessons.
 
 <!--markdownlint-disable-next-line MD034 -->
-> [!VIDEO https://learn-video.azurefd.net/vod/player?show=csharp-101&ep=what-is-c]
+> [!VIDEO https://www.youtube.com/embed/videoseries?si=c1YCsy097ilTxmp3&amp;list=PLdo4fOcmZ0oULFjxrOagaERVAMbmG20Xe]
```

Comment:
I didn't think that we were permitted to link to YouTube in docs, since I believe China will not be able to access it. Are they also hosted on the learn video content site?

Reply:
Yes they are! Let me update that.

Reply:
I don't believe we have that capability, at least not that I'm aware of. But I did just double-check the official docs guidance, and it must have been updated since I looked last. It seems as though YouTube links are fine, but have to be properly embedded using this technique:

https://review.learn.microsoft.com/en-us/help/platform/contribute-video-embed?branch=main#youtube-example

Reply:
Thanks for looking into this, @IEvangelist! I committed the suggested change from your review.